zml/context.zig

```diff
@@ -98,7 +98,11 @@ pub const Context = struct {
                 num_platforms += 1;
             }
         }
-        if (num_platforms == 0) return error.NotFound;
+        if (num_platforms == 0) {
+            log.err("Zero platform available", .{});
+            return error.NotFound;
```

Comment:
arguably NotFound is not the best name. Maybe "NoPlatformAvailable" ?

Reply:
sure I change that, sometimes I forgot the majesty of custom errors and error union in Zig